### PR TITLE
Rtb update

### PR DESCRIPTION
**Pull request by TomLayson**
Cleaned up table display for new Edge section

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
Hello @TomLayson. Thank you for this contribution. Can you resolve the merge conflicts to proceed? Have a nice day.

**Comment by siosulli** [comment]
Just putting do not merge as I want to run a few updates past @TomLayson

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -594,42 +598,30 @@ Alternatively, you can configure the following Registry keys as described:
 
 ### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise
 
+For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies).
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:601]:
```suggestion
For a complete list of the Microsoft Edge policies, see [Group Policy and Mobile Device Management (MDM) settings for Microsoft Edge](/microsoft-edge/deploy/available-policies). 
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 2]
```diff
@@ -594,42 +598,30 @@ Alternatively, you can configure the following Registry keys as described:
 
 ### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise
 
+For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies). 
+
 > [!Important]
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:603]:
```suggestion
> [!IMPORTANT]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**